runtime-extension-model/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -1122,6 +1123,12 @@ private void declareConfiguration(ExtensionDeclarer extensionDeclarer) {
         .describedAs("The default correlation id generation expression for every source. This must be DataWeave expression.");
   }
 
+  private void declareExtraAppConfigs(ConstructDeclarer configuration) {
```

Comment:
rename

Reply:
done, also added a comment to be clearer on what does it represent